homeassistant/components/python_script/__init__.py

```diff
@@ -9,8 +9,10 @@
 
 from homeassistant.const import SERVICE_RELOAD
 from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.service import SERVICE_DESCRIPTION_CACHE
```

Comment:
no no, don't import the constant. Create a function inside `helpers.service`. Things inside `hass.data` are owned by the file and are not allowed to be directly edited.

Note, we will also need tests for this.